helm-rhythmbox.el

```diff
@@ -120,6 +120,15 @@ formatted with `helm-rhythmbox-candidate-format'."
       (dbus-call-method :session service path interface
                         "AddToQueue" (rhythmbox-song-uri song)))))
 
+(defun helm-rhythmbox-pause-song ()
```

Comment:
I should mention that I'm not much of an elisp hacker, so stylistically your suggestion may be more appropriate.